flink-core/src/main/java/org/apache/flink/util/FileUtils.java

```diff
@@ -258,7 +261,82 @@ public static boolean deletePathIfEmpty(FileSystem fileSystem, Path path) throws
 			return false;
 		}
 	}
-	
+
+	/**
+	 * Check whether the two given filesystem is the same or not
+	 *
+	 * @param srcFs
+	 * @param destFs
+	 * @return
+	 */
+	public static boolean compareFs(FileSystem srcFs, FileSystem destFs) {
```

Comment:
Here I just want to check whether the file system is the same or not, if we provide a function compare two URIs, we need to compare more extra elements.